compiler/src/dmd/expressionsem.d

```diff
@@ -13307,8 +13315,10 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
         }
 
         if (exp.e1.type.toBasetype().ty == Taarray)
+        {
             semanticTypeInfo(sc, exp.e1.type.toBasetype());
-
+            getTypeInfoType(exp.loc, exp.e1.type, sc);
```

Comment:
Is this needed? The `semanticTypeInfo()` call above should already generate it - for its `toBasetype()` though.

Reply:
Good catch. Seems like a remainder of early attempts to find where the type info generation was skipped (semanticTypeInfo has some additioonal checks to avoid emission). Removed.